src/ape_compile/__init__.py

```diff
@@ -1,17 +1,18 @@
 from ape import plugins
-from ape.api import ConfigEnum, PluginConfig
-
-
-class EvmVersion(ConfigEnum):
-    constantinople = "constantinople"
-    istanbul = "istanbul"
+from ape.api import PluginConfig
 
 
 class Config(PluginConfig):
-    evm_version: EvmVersion = EvmVersion.istanbul
```

Comment:
Would this break someone's config if trhey had it defined for whatever reason?

Reply:
i made it not break and give a warning and we can fully remove in 0.7